apps/pie-docs/src/assets/styles/components/_contentLayout.scss

```diff
@@ -13,6 +13,18 @@
       flex: 0 1 calc(50% - (var(--dt-spacing-e) / 2));
     }
   }
+
+
+  &.c-columns--three {
+    @include f.media('>=huge') {
+      flex-flow: row wrap;
+
+      > * {
+        // stylelint-disable-next-line declaration-block-no-redundant-longhand-properties
+        flex: 0 1 calc(33.333% - var(--dt-spacing-e)); // Three equal-width columns
+      }
+    }
+  }
 }
```

Comment:
I don't think this is ideal. We should be writing the CSS more fluidly so it renders 3 items on a single row. For example in a grid: `grid-template-rows: repeat(1fr, 3);`.

Why do we specifically need to write separate code for 3 items? 

If I have 2 or 3, but tell the container to display then on a row, why is it different for 3?

Furthermore, what if we want to display 4 or 5 in future?